<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Anachronox .DAT File Extractor</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>571</width>
     <height>72</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btn_select_output_folder">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select Folder for Ouptut of Extracted Files</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_output_folder">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListView" name="lst_files">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>991</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>681</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="btn_select_dat_file">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select DAT File (Confirmation on DAT file will commence extraction to the selected folder)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_dat_file">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_extract_all">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Extract ALL Files</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
